### PR TITLE
To Do list: list structure

### PR DESCRIPTION
**Pull request by brytebee**
## Add to list
In this branch, I implemented a function that takes input from the user and adds this input to a list. This was implemented using 
  * Vanilla Javascript ES6
  * Arrays
  * Arrow functions
  * DOM manipulation
  * Webpack

### CODE REVIEW

File: webpack/package.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
```
Comment (rloterh) [inline webpack/package.json]: - [x] _Webpack_  and other _node packages_ are not installing correctly. Kindly ensure to place the `package.json`,  `package-lock.json` and  `webpack-config.js` are found in the **root** directory of the project

File: webpack/src/index.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import './style.css';
+
+const itemList = [];
+const storeList = document.getElementById('listItem');
+
+// let removeIndex = storeList.children;
+
+const showItem = () => {
```
Comment (rloterh) [inline webpack/src/index.js]: - [x] Your _js_ module does not include `array object` of `tasks`. Kindly ensure that it has an array of some simple to do tasks (array of objects) with each object containing three keys - `description [string]`, `completed [bool]`, `index: [number]`. 
- [x] Also, please ensure that on _page load_, the task array is rendered **dynamically** to the DOM. The tasks should appear in order of the `index` values for each task

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Bright, 

You did extremely great on this project!👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there!👍 

## Good Points 👍  
- No linter errors


## Required Changes ♻️
- [x] _Kindly check the comments under the review._
<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: webpack/src/index.js (hunk 2 above)
Comment (brytebee) [reply webpack/src/index.js]: Thanks for the review. It took a whole day of study to implement though 😩😩

File: package.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "to-do",
+  "version": "1.0.0",
+  "description": "",
+  "main": ".eslintrc.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "Webpack",
+    "watch": "webpack --watch"
+  },
```
Comment (abruzy) [inline package.json]: - [x] Kindly add the start script to your `package.json` file.
- [x] I wasn't able to run the script or start the project as this is not included.

**Review by abruzy — CHANGES_REQUESTED** [review]
Hi @brytebee 👋🏽, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by brytebee** [comment]
Hi. Could you help me out?
I added the `start ` script and I'm getting all sorts of errors I can't debug.

 * A code I tried
 `  "scripts": {
    "start": "node ./dist/main.js",
    "build": "Webpack",
    "watch": "webpack --watch",
    "test": "echo \"Error: no test specified\" && exit 1"
  },`

 * current code
 `  "scripts": {
    "start": "npm run build",
    "build": "Webpack",
    "watch": "webpack --watch",
    "test": "echo \"Error: no test specified\" && exit 1"
  },`

**Review by dasileker — APPROVED** [review]
### **Hi @brytebee** 
### **Project Approved ✅**

### **Optional**

- Please stick with the wireframe, your project is having colors that is different.

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**